docs/style.css

```diff
@@ -1,3 +1,28 @@
+/* Colors */
+
```

Comment:
Nitpick: dat is wat mij betreft de extra bytes en schermruimte niet waard, uit de backets/opmaak/code zelf is dit al wel duidelijk?

```suggestion
```

Reply:
Eens, heb al deze comments verwijderd.